enable tests/drivers/adc/adc_dma on board for nucleo_h743zi

Desvauxm-st opened the pull request:
the twister test is now enable for tests/drivers/adc/adc_dma on board
 nucleo_h743zi.   
 add platform_allow nucleo_h743zi.
 used on driver st_stm32_adc with CONFIG_ADC_STM32_DMA=y.
     
 Signed-off-by: Marc Desvaux <marc.desvaux-ext@st.com>

Desvauxm-st committed:
boards: arm: nucleo_h743zi: add dma supported

add dma supported on board nucleo_h743zi.
the twister test is now enable for tests/drivers/adc/adc_dma on board
nucleo_h743zi.
used on driver st_stm32_adc with CONFIG_ADC_STM32_DMA=y.


Signed-off-by: Marc Desvaux <marc.desvaux-ext@st.com>

erwango reviewed (changes requested):
I guess a board overlay is required to get the test passed

erwango commented:
> in fact the nucleo_h743zi.overlay and nucleo_h743zi.conf already exist into tests/drivers/adc/adc_dma/board

Sorry, I was searching on wrong branch. Approved

Can you rebase ?

Desvauxm-st committed:
tests: drivers: adc: adc_dma: add nucleo_h743zi

add platform_allow nucleo_h743zi.
the twister test is now enable for tests/drivers/adc/adc_dma on board
nucleo_h743zi.
used on driver st_stm32_adc with CONFIG_ADC_STM32_DMA =y.

Signed-off-by: Marc Desvaux <marc.desvaux-ext@st.com>
